src/pages/CareerPage/CareerPageInner.tsx

```diff
@@ -0,0 +1,142 @@
+import { Col, Container, Row } from "react-bootstrap";
+import "./CareerPage.scss";
+import { ButtonComponent } from "../../components/button/ButtonComponent";
+import SvgButton from "../../assets/icons/button.svg"
+import { CTAcomponent } from "../../components/ctaSection/CTAcomponent";
+import {NavLink} from "react-router-dom";
+import { useState } from "react";
+
+
+export const CareerPageInner = () => {
+    const [details, setDetails] = useState(true);
+    const [requirements, setRequirements] = useState(false);
+    const [responsibilites, setResponsibilities] = useState(false);
```

Comment:
These are boolean values. 
So ideally these should be name

const [showRequirements, setShowRequirements] = useState(false);

similar for responsibilities and details